modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -8,47 +10,56 @@ import edu.gemini.spModel.seqcomp.SeqConfigNames._
 
 import scalaz.Scalaz._
 import scalaz._
+import scalaz.concurrent.Task
 
 /**
   * Created by jluhrs on 9/14/16.
   */
 object SeqTranslate {
 
+  case class Systems(
+                    dhs: DhsClient,
+                    tcs: TcsController,
+                    flamingos2: Flamingos2Controller
+                    )
+
+  private val dhsSimulator = DhsClientSim(LocalDate.now)
```

Comment:
Yes, but it might not be worth introducing a `Task` with this PR. With a `TODO` comment to not forget about it in the future would be good enough.